kamelet-support/src/main/java/io/kaoto/backend/KamelPopulator.java

```diff
@@ -136,6 +136,8 @@ public void populateKamelet(
         setSpecDependencies(kamelet.getSpec(), steps, metadata);
         setSpecDefinition(kamelet, parameters);
         kamelet.getSpec().getTemplate().setBeans((List<Bean>) metadata.getOrDefault("beans", null));
+        final String description = metadata.get("description").toString();
```

Comment:
Add a nullpointer check here. If description is null, it fails (as in the tests)

Reply:
hmm "funny" is that I had it locally I've just haven't pushed it. 
 `final String description = metadata.getOrDefault("description","").toString();`